bin/bump-version.rb

```diff
@@ -52,7 +52,7 @@
 puts "git push origin HEAD:v#{new_version}-release-notes"
 puts "# ... create PR, verify, merge, for example:"
 puts "gh pr create"
-puts "git pull"
-puts "git tag 'v#{new_version}'"
-puts "git push --tags origin main"
+puts "# tag the approved release notes:"
+puts "git tag 'v#{new_version}' 'v#{new_version}-release-notes'"
```

Comment:
```suggestion
puts "git fetch"
puts "git tag 'v#{new_version}' 'origin/v#{new_version}-release-notes'"
```

This seemed right enough when I wrote it, but because of :52 this ref doesn't exist locally. I think referencing the remote branch is the best idea to keep this as something you can cleanly paste.

I also figured `git pull` wasn't necessary, but maybe the release notes were appended during review (e.g. via a suggestion comment like this!).